### DISCUSSION TITLE
Profile Verifications - the rel=me link is not picked up

### DISCUSSION THREAD

**Discussion by jshute3** [mastodon/mastodon#20354] [Q&A] [answered]
Two questions
1) Does the process doing the searching for the rel=me link follow a 301 code and jump to the location provided?
```
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="https://sites.google.com/a/shute.org/www/">here</A>.
</BODY></HTML>
```

2) How sensitive/robust is the search algorithm. I have the link provided in the edit profile page on sites.google.com hosting site and the link looks pretty mangled in the html pulled by curl.  It looks like an input to another call/process. Not a HTMl coder.

`data-code="&lt;a` rel=&quot;me&quot; href=&quot;https://twit.social/@Jshute&quot;&gt;Mastodon&lt;/a&gt;"`

**Comment by tathamoddie** [comment, marked as answer]
I don't believe that you'll be able to complete this with Google Sites.

Google Sites puts the markup within an `IFRAME`, and encodes the original raw response (what you're seeing as 'mangled'). This is probably an intentional control on the Google Sites side as they need to be confident that their platform isn't used to host malicious code on a `*.google.com` domain.

I had a look at your site (https://sites.google.com/a/shute.org/www/) and confirm that it won't validate, with the way that Google is processing and returning the markup.

I tried a few approaches of spinning this up on my own Google Site test quickly, but all routes led to the same encoding.

**Reply by jshute3** [reply]
Thank you for taking a look. I had a suspicion that this was the case. Google Sites is what I have for web sites for now (holding on to the early adopter free deal).
